src/main/java/org/apache/datasketches/theta/SingleItemSketch.java

```diff
@@ -99,6 +101,21 @@ public static SingleItemSketch heapify(final Memory srcMem, final long seed) {
     throw new SketchesArgumentException("Input Memory is not a SingleItemSketch.");
   }
 
+  /**
+   * Creates a SingleItemSketch on the heap given a SingleItemSketch Memory image and a seedHash.
+   * Checks the seed hash of the given Memory against a hash of the given seed.
+   * @param srcMem the Memory to be heapified.
+   * @param seedHash a given seedHash
+   * @return a SingleItemSketch
+   */ //does not override Sketch
+  public static SingleItemSketch heapify(final Memory srcMem, final short seedHash) {
```

Comment:
it would probably be clearer here if `seedHash` was renamed `expectedSeedHash` as the actual seed has is extracted from the `srcMem` object.

Reply:
This is a good suggestion.  So I made this variable name change everywhere applicable (see comments on latest push).